tests/foreman/virtwho/test_cli_virtwho.py

```diff
@@ -343,8 +343,7 @@ def test_positive_post_hypervisors(self):
         :BZ: 1637042, 1769680
         """
         data = hypervisor_json_create(hypervisors=100, guests=10)
-        owner = "owner=Default_Organization&env=Library"
-        url = f'https://{settings.server.hostname}/rhsm/hypervisors?{owner}'
+        url = f'https://{settings.server.hostname}/rhsm/hypervisors/Default_Organization'
```

Comment:
Use Default_Org from `robottelo.constants` instead of hardcoding here.

Reply:
the `Default_Org` from `robottelo.constants` is the Name(not label), so I use the org.info to get it.